src/sentry/integrations/github/integration.py

```diff
@@ -155,43 +155,65 @@ def get_document_origin(org) -> str:
 # Github App docs and list of available endpoints
 # https://docs.github.com/en/rest/apps/installations
 # https://docs.github.com/en/rest/overview/endpoints-available-for-github-apps
-class GitHubIntegration(IntegrationInstallation, GitHubIssueBasic, RepositoryMixin, CommitContextMixin):  # type: ignore[misc]
-    repo_search = True
+class GitHubIntegration(IntegrationInstallation, RepositoryIntegration, GitHubIssueBasic, CommitContextMixin):  # type: ignore[misc]
     codeowners_locations = ["CODEOWNERS", ".github/CODEOWNERS", "docs/CODEOWNERS"]
 
+    @property
+    def integration_name(self) -> str:
+        return "github"
+
```

Comment:
everything besides this in this file is the same, i just grouped methods together